src/v/kafka/server/server.cc

```diff
@@ -200,6 +203,20 @@ ss::future<security::tls::mtls_state> get_mtls_principal_state(
       });
 }
 
+/*static*/ std::vector<bool> server::convert_api_names_to_key_bitmap(
```

Comment:
drop the comment?

Reply:
It's on purpose to make static member function definitions more visible. Marked by habit but I think it's useful. LMK if still needs to be dropped.

Reply:
sgtm, no need to drop it. but i'm just curious what practical value it serves? with modern clangd lsp, you can't get very far if you start trying to access class variables.

Reply:
It is helping a reader to understand the context faster. It can be particularily helpful in cases when a static member is called via `object.function()`. Also otherwise, it saves from wondering why the function is not a `const` member and whether it can affect the state of `this`.